libs/tup-components/src/projects/users/UserList/ManageTeam.tsx

```diff
@@ -1,86 +1,33 @@
-import {
-  InlineMessage,
-  LoadingSpinner,
-  Button as CoreButton,
-} from '@tacc/core-components';
-import {
-  useAddProjectUser,
-  useProjectUsers,
-  useProject,
-  useRoleForCurrentUser,
-} from '@tacc/tup-hooks';
-import { useState } from 'react';
-import { Input, InputGroup, Button } from 'reactstrap';
+import { Button } from '@tacc/core-components';
+import { useProject, useRoleForCurrentUser } from '@tacc/tup-hooks';
 import styles from './UserList.module.css';
+import AddUserModal from './AddUserModal';
 
 const ManageTeam: React.FC<{ projectId: number }> = ({ projectId }) => {
-  const { data: users } = useProjectUsers(projectId);
   const currentUserRole = useRoleForCurrentUser(projectId) ?? '';
-  const { mutate, isLoading, error } = useAddProjectUser(projectId);
-  const [userToAdd, setUserToAdd] = useState<string>('');
   const { data: project } = useProject(projectId);
   const chargeCode = project?.chargeCode;
 
   if (!['PI', 'Delegate'].includes(currentUserRole) || !chargeCode) return null;
 
   if (chargeCode.startsWith('TG-')) {
     return (
-      <div className={styles['manage-team']} style={{ paddingBottom: '5px' }}>
+      <div className={styles['add-member-button']}>
```

Comment:
**Semantics** The naming used can easily cause confusion.
- Alternative class name — `add-member-actions` — because this is not a button.
- Alternative class assignment — restore and use `className={styles['access-button']}` for the `add-member-button` styles.

I have not tried either. I do not know what difficulties will arise.

Reply:
I changed the class to `user-navactions` (based off of the existing `user-navitem`) via 66a4215. If that doesn't sound right, feel free to revert.